source/common/http/http1/legacy_parser_impl.cc

```diff
@@ -38,7 +38,7 @@ class LegacyHttpParserImpl::Impl {
     parser_.allow_chunked_length = 1;
   }
 
-  Impl(http_parser_type type, void* data) : Impl(type) {
+  Impl(http_parser_type type, void* data, bool parse_status) : Impl(type) {
```

Comment:
I would just always parse status, the cost is low.

Reply:
Removed the flag.